src/GraphQL/Validation/ValidationOptions.cs

```diff
@@ -22,7 +29,7 @@ namespace GraphQL.Validation
         /// Gets or sets a list of rules to use to validate the <see cref="GraphQLDocument"/>.
         /// If no rules are specified, <see cref="DocumentValidator.CoreRules"/> are used.
         /// </summary>
-        public IEnumerable<IValidationRule>? Rules { get; init; }
+        public IEnumerable<IValidationRule>? Rules { get; init; } = null;
```

Comment:
> Error	CS0843	Auto-implemented property 'ValidationOptions.Rules' must be fully assigned before control is returned to the caller.

Reply:
@Shane32  You may update your .NET SDK to the latest 6.x.x to see all these errors. 6.0.102 works well 😕